.github/workflows/tests.yml

```diff
@@ -0,0 +1,66 @@
+# TODO: Line 20, enable python-version: 3.5
```

Comment:
Given that Py35 reaches end-of-life in less than two weeks, let's just remove support.
https://devguide.python.org/#status-of-python-branches